### PR TITLE
tweaks

### PR DESCRIPTION
**Pull request by SamVerschueren**
Tweaked the code a little

- Added XO (and removed .jshintrc)
- Bumped AVA (and rewritten the tests)
- Wrote documentation in ES2015 style

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

File: package.json [hunk 1]
```diff
@@ -31,14 +31,15 @@
...
+    "test": "xo && ava"
   },
   "dependencies": {
     "number-is-nan": "^1.0.0"
   },
   "devDependencies": {
     "array-equal": "^1.0.0",
-    "ava": "^0.0.4"
+    "ava": "*",
+    "xo": "*"
```
Comment (mshenfield) [inline package.json]: It would be nice to pin these to a specific version,

```js
"ava": "^0.8.0",
"xo": "^0.12.0"
```

Comment (sindresorhus) [inline package.json]: @mshenfield I considered it, but I just have too many modules and don't want to have to bump them all the time. They're my modules anyways so I can update them when I make breaking changes, which will be very rare going forward.

**Comment by SamVerschueren** [comment]
Merged master and fixed merge conflicts.

File: index.js [hunk 2]
```diff
@@ -4,13 +4,13 @@ var numberIsNan = require('number-is-nan');
...
-function match(arr, value){
-	return arr.some(function(toMatch) {
-		return toMatch instanceof RegExp
-			? toMatch.test(value)
-			: toMatch === value;
+function match(arr, value) {
+	return arr.some(function (toMatch) {
+		return toMatch instanceof RegExp ?
+			toMatch.test(value) :
+			toMatch === value;
```
Comment (SamVerschueren) [inline index.js]: @sindresorhus Do you prefer one line or multi-line?

Comment (sindresorhus) [inline index.js]: One line if it's not too long. In this case one line.

**Comment by sindresorhus** [comment]
Thank you :)

**Comment by SamVerschueren** [comment]
No problem :)

**Pull request closed without merging**